config-jest/setup/shims.js

```diff
@@ -1 +1,6 @@
 require('airbnb-js-shims/target/es2019');
+
+const { TextEncoder, TextDecoder } = require('util');
+
+global.TextEncoder = TextEncoder;
+global.TextDecoder = TextDecoder;
```

Comment:
I've had to add this because `jest-environment-jsdom` does not contain `TextEncoder` and `TextDecorder`, but `react` requires them.